### PR TITLE
CHANGELOG: release notes for v3.0.0-rc.1,v2.1.9,v2.1.8

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

New added changelog for v3.0.0-rc.1, and cherry-pick v2.1.8,v2.1.9 change logs

@zz-jason @jackysp @shenli @morgo PTAL

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,104 @@
...
+* Specially handle `GROUP_CONCAT` when eliminating aggregate functions during the logical optimization phase to prevent an incorrect result [#9967](https://github.com/pingcap/tidb/pull/9967)
+* Properly push the filter down to child nodes of the join operator if the filter is a constant [#9848](https://github.com/pingcap/tidb/pull/9848)
+* Specially handle some functions such as `RAND()` when pruning columns during the logical optimization phase to prevent a result with an incompatibility with MySQL [# 10064](https://github.com/pingcap/tidb/pull/10064)
+* Support `FAST ANALYZE`, which speeds up statistics collection by sampling the region instead of scanning the entire region. This feature is disabled by default. To enable it, set the variable `tidb_enable_fast_analyze` to true [#10258](https://github.com/pingcap/tidb/pull/10258)
+* Support `SQL PLAN MANAGEMENT`, which ensures execution stability by performing execution plan binding on SQL statements. This feature is currently in beta and only supports using bound execution plans for SELECT statements. It is not recommended to use it in the production environment. [#10284](https://github.com/pingcap/tidb/pull/10284)
```
Comment (morgo) [inline CHANGELOG.md]:
```suggestion
* Support SQL Plan Management, which ensures execution stability by performing execution plan binding on SQL statements. This feature is currently in beta and only supports using bound execution plans for SELECT statements. It is not recommended to use it in the production environment. [#10284](https://github.com/pingcap/tidb/pull/10284)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,104 @@
...
+### Execution Engine
+* Support tracing and controlling memory usage in three operators - `TableReader`, `IndexReader` and `IndexLookupReader` [#10003](https://github.com/pingcap/tidb/pull/10003)
+* Support showing more information about COPROCESSOR tasks in the slow log such as the number of tasks in COPROCESSOR, the average/longest/90% of execution/waiting time and the longest execution/waiting time of the TiKV address [#10165](https://github.com/pingcap/tidb/pull/10165)
```
Comment (morgo) [inline CHANGELOG.md]:
```suggestion
* Support showing more information about coprocessor tasks in the slow log such as the number of tasks in coprocessor, the average/longest/90% of execution/waiting time and the longest execution/waiting time of the TiKV address [#10165](https://github.com/pingcap/tidb/pull/10165)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,104 @@
...
+* Support the prepared DDL statements with no placeholders
+[#10144](https://github.com/pingcap/tidb/pull/10144)
+### Server
+* Only allow the DDL owner to execute bootstrap when TiDB gets started
+[#10029](https://github.com/pingcap/tidb/pull/10029)
+* Add the variable `tidb_skip_isolation_level_check` to prevent TiDB from reporting errors when setting the transaction isolation level to SERIALIZABLE
+[#10065](https://github.com/pingcap/tidb/pull/10065)
+* Merge the implicit commit time and the SQL execution time in the slow log
+[#10294](https://github.com/pingcap/tidb/pull/10294)
+* RBAC Privilege Management
```
Comment (morgo) [inline CHANGELOG.md]:
```suggestion
* Support for SQL Roles (RBAC Privilege Management)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,104 @@
...
+* Support `FAST ANALYZE`, which speeds up statistics collection by sampling the region instead of scanning the entire region. This feature is disabled by default. To enable it, set the variable `tidb_enable_fast_analyze` to true [#10258](https://github.com/pingcap/tidb/pull/10258)
+* Support SQL Plan Management, which ensures execution stability by performing execution plan binding on SQL statements. This feature is currently in beta and only supports using bound execution plans for SELECT statements. It is not recommended to use it in the production environment. [#10284](https://github.com/pingcap/tidb/pull/10284)
+### Execution Engine
+* Support tracing and controlling memory usage in three operators - `TableReader`, `IndexReader` and `IndexLookupReader` [#10003](https://github.com/pingcap/tidb/pull/10003)
+* Support showing more information about coprocessor tasks in the slow log such as the number of tasks in coprocessor, the average/longest/90% of execution/waiting time and the longest execution/waiting time of the TiKV address [#10165](https://github.com/pingcap/tidb/pull/10165)
```
Comment (XuHuaiyu) [inline CHANGELOG.md]: `the longest execution/waiting time of the TiKV address` is not correct.

It's `the addresses of the TiKVs which take the longest execution time or waiting time`

Comment (XuHuaiyu) [reply CHANGELOG.md]: PTAL @lilin90

Comment (lilin90) [reply CHANGELOG.md]: @yikeke Please take a look. Thanks!

**Pull request merged**